server/repository/src/db_diesel/assets/asset_catalogue_item_property.rs

```diff
@@ -38,8 +51,24 @@ impl<'a> AssetCatalogueItemPropertyPropertyRepository<'a> {
 
         Ok(result.into_iter().map(to_domain).collect())
     }
-}
 
+    pub fn query_property_and_value(
```

Comment:
this is a convenience - I'm wanting to show property name (from `asset_catalog_property` table) and the assigned value for this asset (which is in the `asset_catalog_item_property` table) and rather than making two database calls, I'm doing this so that I can use the inner join.